### PR TITLE
Fix internal links

### PR DESCRIPTION
**Pull request by MeBrei**
Screenshot 2020-11-24 at 13 55 37

Due to the leading `/` the links could not be resolved and clicking on the links lead to an error page

### CODE REVIEW

**Comment by rudolfs** [comment]
We should make the link formatting consistent with the other docs. `[link title][lt]` and then list `[lt]: ...` at the bottom of the page.

File: docs/understanding-radicle/glossary.md [hunk 1]
```diff
@@ -46,13 +46,13 @@ to disseminate information. Participants in the Radicle network share and spread
 data by keeping redundant copies of projects locally and sharing updates with
 peers. This allows data to be disseminated according to use and value: the more
 peers who are interested in certain data the more available it is
-made to the network. See [Gossip](/how-it-works.md/#replication-model)
+made to the network. See [Gossip](how-it-works.md/#replication-model)
 
 ## maintainer
-A [user][us] who manages updates to the canonical view of a project. A
+AAA [user][us] who manages updates to the canonical view of a project. A
```
Comment (rudolfs) [inline docs/understanding-radicle/glossary.md:52]: `AAA` 👀

Comment (MeBrei) [reply docs/understanding-radicle/glossary.md:52]: 🤦

File: docs/understanding-radicle/glossary.md [hunk 2]
```diff
@@ -77,13 +77,13 @@ A device running the Radicle Link protocol.
 ## Device ID
 The encoding of a peer's public key tied to their device. In the Beta, there will
 only be one Device ID per user (See
-[Architecture](how-it-works.md/#architecture))
+[Architecture][hiw-ar])
```
Comment (MeBrei) [inline docs/understanding-radicle/glossary.md:80]: This link/heading does not exist. Is it supposed to link to [Key Architecture](http://localhost:3000/docs/understanding-radicle/how-it-works/#key-architecture)?

Comment (rudolfs) [reply docs/understanding-radicle/glossary.md:80]: Probably.

Comment (abbey-titcomb) [reply docs/understanding-radicle/glossary.md:80]: No, we're deleting this section. Will have to update it

**Comment by abbey-titcomb** [comment]
Ahhhhh, I had made all of these fixes, just haven't pushed them yet 😅 Maybe we should merge this then I can pull down and make sure all my changes are captured as well?

File: docs/understanding-radicle/glossary.md [hunk 3]
```diff
@@ -142,10 +142,19 @@ Upstream app.
...
 [br]: #branch
+[ma]: #maintainer
 [pe]: #peer
 [pr]: #project
 [re]: #repository
 [ri]: #radicle-id
 [us]: #user
 
+[hiw]: how-it-works.md
+[hiw-ar]: how-it-works.md/#architecture
```
Comment (abbey-titcomb) [inline docs/understanding-radicle/glossary.md:153]:
```suggestion
[hiw-ar]: how-it-works.md/#data-model
```

**Review by abbey-titcomb — CHANGES_REQUESTED** [review]
lgtm!

**Review by rudolfs — APPROVED** [review]

**Review by abbey-titcomb — APPROVED** [review]
🎉🎉🎉🎉🎉🎉

**Pull request merged**